dbt-snowflake/src/dbt/include/snowflake/macros/relations/table/create.sql

```diff
@@ -1,69 +1,29 @@
 {% macro snowflake__create_table_as(temporary, relation, compiled_code, language='sql') -%}
 
-  {%- if relation.is_iceberg_format and not adapter.behavior.enable_iceberg_materializations.no_warn %}
-    {% do exceptions.raise_compiler_error('Was unable to create model as Iceberg Table Format. Please set the `enable_iceberg_materializations` behavior flag to True in your dbt_project.yml. For more information, go to https://docs.getdbt.com/reference/resource-configs/snowflake-configs#iceberg-table-format') %}
-  {%- endif %}
-
-  {%- set materialization_prefix = relation.get_ddl_prefix_for_create(config.model.config, temporary) -%}
-  {%- set alter_prefix = relation.get_ddl_prefix_for_alter() -%}
-
-  {# Generate DDL/DML #}
-  {%- if language == 'sql' -%}
-    {%- set cluster_by_keys = config.get('cluster_by', default=none) -%}
-    {%- set enable_automatic_clustering = config.get('automatic_clustering', default=false) -%}
-    {%- set copy_grants = config.get('copy_grants', default=false) -%}
-
-    {%- if cluster_by_keys is not none and cluster_by_keys is string -%}
-      {%- set cluster_by_keys = [cluster_by_keys] -%}
+{%- if relation.is_iceberg_format and not adapter.behavior.enable_iceberg_materializations.no_warn -%}
+    {%- do exceptions.raise_compiler_error('Was unable to create model as Iceberg Table Format. Please set the `enable_iceberg_materializations` behavior flag to True in your dbt_project.yml. For more information, go to https://docs.getdbt.com/reference/resource-configs/snowflake-configs#iceberg-table-format') -%}
+{%- endif -%}
+
+{%- set catalog_name = config.get('catalog_name', default=none) -%}
+{%- if catalog_name is not none -%}
+    {%- set catalog_integration = adapter.get_catalog_integration(catalog_name) -%}
+{%- else -%}
+    {%- set catalog_integration = none -%}
+{%- endif -%}
+
+{%- if language == 'sql' -%}
+    {%- if catalog_integration is none -%}
```

Comment:
The only way we'll have catalog integration is if the user opted-in to this feature. We will migrate the managed catalog over later.